packages/patternfly-4/react-charts/src/components/ChartTheme/themes/theme-color.js

```diff
@@ -45,6 +46,9 @@ export default props => {
     bar: {
       colorScale: COLOR_SCALE,
       style: {
+        data: {
+          fill: COLOR_BAR_FILL
+        },
```

Comment:
Instead of providing a separate COLOR_BAR_FILL var, my thought was to use the first color defined by the color scale.

Reply:
Ultimately, colors will come from pf-core via react-tokens. I'm working to make react-tokens available now.

For example, here is a snippet of colors defined for the bar chart below. 

![Screen Shot 2019-05-29 at 4 21 05 PM](https://user-images.githubusercontent.com/17481322/58588596-007fe900-822e-11e9-905a-0e7f9a2466d9.png)

Reply:
Note that these colors are ordered.

Reply:
Yeah, that was my original implementation. I can change back to that if that's what we prefer.

Reply:
Yes, please. I'd like the default color to come from the ordered colorScale defined for each chart.